packages/safe-ds-vscode/src/extension/mainClient.ts

```diff
@@ -137,12 +141,177 @@ const registerMessageLoggingCallbacks = function () {
 };
 
 const registerVSCodeCommands = function (context: vscode.ExtensionContext) {
+    const registerCommandWithCheck = (commandId: string, callback: (...args: any[]) => any) => {
+        return vscode.commands.registerCommand(commandId, (...args: any[]) => {
+            if (!isPythonServerAvailable()) {
+                vscode.window.showErrorMessage('Extension not fully started yet.');
+                return;
+            }
+            return callback(...args);
+        });
+    };
+
     context.subscriptions.push(
         vscode.commands.registerCommand('extension.safe-ds.runPipelineFile', commandRunPipelineFile),
     );
+
+    // Not used right now
+    // context.subscriptions.push(
+    //     registerCommandWithCheck("eda-test01.runEda", () => {
+    //         EDAPanel.createOrShow(context.extensionUri, context, "", undefined, pythonServerPort);
+    //     }),
+    // );
+
+    // context.subscriptions.push(
+    //     registerCommandWithCheck("eda-test01.runEdaFromTable", () => {
+    //         const { activeTextEditor } = vscode.window;
+
+    //         if (!activeTextEditor) {
+    //         vscode.window.showErrorMessage("No ative text editor!");
+    //         return;
+    //         }
+
+    //         const newtableIdentifier = activeTextEditor.document.getText(activeTextEditor.selection);
+
+    //         if (newtableIdentifier.trim() === "") {
+    //         vscode.window.showErrorMessage("No text selected!");
+    //         return;
+    //         }
+
+    //         EDAPanel.createOrShow(context.extensionUri, context, "", newtableIdentifier, pythonServerPort);
+    //     }),
+    // );
+
+    context.subscriptions.push(
+        registerCommandWithCheck('eda-test01.runEdaFromContext', () => {
```

Comment:
Will need to be updated accordingly once the command is changed in `package.json`.